utility/packed_file_info.h

```diff
@@ -76,58 +81,56 @@ class PackedFileInfo : public RefCounted {
 		}
 		while (path.begins_with("/")) {
 			path = path.substr(1, path.length() - 1);
-			malformed_path = true;
 		}
-		while (path.find("...") >= 0) {
```

Comment:
The reason for this function is that the Godot resource loaders choke on paths that contain the items that we're replacing, and the Godot project exporter will never export resources into a PCK with paths containing these. The logic of this shouldn't be changed unless you're certain that paths containing the items that you've removed are considered valid paths by the loader.

Reply:
But I see the game where paths literally contain `...` in the image names so the Godot put them there in the first place and no chocking occurs on game load. How is this possible if they are not valid?

Reply:
If it's version related stuff or depends on some non-trivial logic, please add the fixes to this in a later commits because I neither don't know the codebase, nor want to learn it for just a couple of games...